src/vrad.py

```diff
@@ -152,27 +153,38 @@ async def main(argv: List[str]) -> None:
         # Detect preview via knowing the bsp name. If we are in preview,
         # check the config file to see what was specified there.
         if os.path.basename(path) == "preview.bsp":
-            edit_args = not config.get_bool('General', 'vrad_force_full')
+            # Checks what the light config was set to.
+            light_args = config.get_val('General', 'vrad_compile_type')
```

Comment:
Since this is a string not a bool (which automatically does validation), this should probably check that if the value is valid, and if not default to `FAST` - if it's invalid right not it'd default to no VRAD which isn't exactly desirable.